packages/neptune-extension/src/common/hooks/useSelectInputValue.ts

```diff
@@ -59,12 +58,10 @@ export default function useSelectInputValue(
     options,
     onChange,
   };
-
   const metaProps: SelectMetaProps = {
-    valid,
-    loading,
+    status: loading ? 'pending' : valid ? 'success' : 'error',
```

Comment:
Would you mind working on this, because I guess such use of the ternary operator is discouraged by most js guidelines.